config/routes.rb

```diff
@@ -2,5 +2,5 @@
 
 Rails.application.routes.draw do
   devise_for :users
-  root to: 'home#index'
+  root to: 'users#index'
```

Comment:
You don't have users controller now => an error on root page. Remove this line for now